### PR TITLE
Changelog 4.28

### PR DESCRIPTION
**Pull request by pmaytak**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+**Fixed `System.InvalidOperationException` when calling `GetAccountAsync` in a Xamarin Android app**. See issue [#2434](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2434) for details.  
+**Fixed a bug when a WAM account picker window was not correctly parented to windows**. See issue [#2469](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2469) for details.  
+**Fixed the behavior of `WithAuthority(string)` to correctly parse an authority string**. See [#2412](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2412) for details.  
+**Improved .NET 5 support for older versions of Windows**. See issue [#2445](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2445) for details.  
+**Added MSAL.NET assembly to `rd.xml` to enable MSAL.NET to work in optimized UWP apps**. See issue [#1617](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1617) for details.
+
+### Fundamentals:
+**Added additional code analyzers**. See issue [#2419](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2419) for details.
+
```
Comment (pmaytak) [inline changelog.txt:25]: @trwalke Not sure what to put for #2139.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+**Added support for specifying a custom fixed version of WebView2 runtime**. See issue [#2446](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2446) for details.  
+**Added helper methods for desktop apps**. See issue [#2459](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2459) for details.  
+**Added `refresh_in` logic to On-Behalf-Of flows**. See issue [#2389](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2389) for details.  
+**Stopped using reflection to deserialize JSON** to improve Unity apps built for UWP. See issue [#2343](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2343) and [Troubleshooting Unity](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Troubleshooting-Unity) for details.
+
+### Enhancements:
+**Added additional logging when the cache is not serialized in confidential client apps**. See issue [#2461](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2461) and [Token cache serialization](https://aka.ms/msal-net-cca-token-cache-serialization) for details.
```
Comment (jmprieur) [inline changelog.txt:13]:
```suggestion
**Added additional logging when the cache is not serialized in confidential client apps** to help choosing the right token cache serialization. See issue [#2461](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2461) and [Token cache serialization](https://aka.ms/msal-net-cca-token-cache-serialization) for details.  
```

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+**Added helper methods for desktop apps**. See issue [#2459](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2459) for details.  
+**Added `refresh_in` logic to On-Behalf-Of flows**. See issue [#2389](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2389) for details.  
+**Stopped using reflection to deserialize JSON** to improve Unity apps built for UWP. See issue [#2343](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2343) and [Troubleshooting Unity](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Troubleshooting-Unity) for details.
+
+### Enhancements:
+**Added additional logging when the cache is not serialized in confidential client apps**. See issue [#2461](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2461) and [Token cache serialization](https://aka.ms/msal-net-cca-token-cache-serialization) for details.  
+**`GetAccountsAsync()` is now obsolete in confidential client apps**. Use `GetAccountAsync(string)`. See issue [#1967](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1967) for details.
```
Comment (jmprieur) [inline changelog.txt:14]:
```suggestion
**`GetAccountsAsync()` is now obsolete in confidential client apps** as confidential client applications need to have one cache per account. Use `GetAccountAsync(string)`. See issue [#1967](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1967) for details.  
```

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,28 @@
+4.28.0
+==========
+### New Features:
+**Updated token cache related telemetry**. See issue [#2406](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2406) for details.  
+**Added support for WebView2**. See issue [#1398](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1398) and [WebView2 wiki](https://aka.ms/msal-net-webview2.) for details.  
+**Added the ability to set a window title of a WebView2 window**. See issue [#2397](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2397) for details.  
+**Added support for specifying a custom fixed version of WebView2 runtime**. See issue [#2446](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2446) for details.  
+**Added helper methods for desktop apps**. See issue [#2459](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2459) for details.  
+**Added `refresh_in` logic to On-Behalf-Of flows**. See issue [#2389](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2389) for details.
```
Comment (jmprieur) [inline changelog.txt:9]:
```suggestion
**Added `refresh_in` logic to On-Behalf-Of flows** as was the other flows, to improve resilence. See issue [#2389](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2389) for details.  
```

File: changelog.txt (hunk 1 above)
Comment (jmprieur) [reply changelog.txt:25]: @pmaytak : what about improved documentation to support Android 11 ? See https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-net-xamarin-android-considerations#android-11-support ?

Comment (pmaytak) [reply changelog.txt:26]: Added. Thanks.

**Pull request merged**